contrib/kcp/README.md

```
@@ -121,27 +149,53 @@ kubectl ws use :root
 
-export KUBECONFIG=.kcp/consumer2.kubeconfig
-go run ./cmd/konnector/ --lease-namespace default --server-address :8091
+./bin/konnector --lease-namespace default --kubeconfig .kcp/consumer2.kubeconfig --server-address :8091
 ```
 
+This will keep running, so switch to a new terminal.
+
+## Testing
+
 Create objects:
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)